src/main/java/com/saintsrobotics/swerveDrive/tasks/teleop/ArmsTask.java

```diff
@@ -20,7 +20,8 @@
     // Find actual values later
     private double fullInPosition = 300;
```

Comment:
Looks like these 3 are being used as constants, can you make them `private static final double` instead?